src/Explorer/Panes/PublishNotebookPane/PublishNotebookPane.tsx

```diff
@@ -105,7 +105,6 @@ export const PublishNotebookPane: FunctionComponent<PublishNotebookPaneAProps> =
         notebookName,
         notebookDescription,
         notebookTags?.split(","),
-        author,
```

Comment:
Remove all references to this variable

Reply:
this is also needed in the DE for the card preview, the references that exist are all used in that context